servers.json

```diff
@@ -1524,3 +1531,4 @@
         ]
     }
 ]
+
```

Comment:
Please remove this blank line. Also, the file name of your logo must match this format `<id>.png`. You currently have `nytecc.png`, it must be `nyte.png`.

Reply:
done